src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static final int MAX_BALL_NUMBER_RANDOM = 100;
+
+    public Ball getRandomBall() {
+        return new Ball(new Random().nextInt(MAX_BALL_NUMBER_RANDOM),
+                        new ColorSupplier().getRandomColor());
```

Comment:
the same as previous comment.
should we create new instance of Random and ColorSupplier  every time when the method is called?
create a class attributes for  Random and ColorSupplier and initialize them at class level, not inside the method block.